README.md

```diff
@@ -147,3 +147,23 @@ https://geekhack.org/index.php?topic=107316.0
 - **ai03** for [MX_Alps_Hybrid](https://github.com/ai03-2725/MX_Alps_Hybrid) and [random-keyboard-parts.pretty](https://github.com/ai03-2725/random-keyboard-parts.pretty). Also got a lot of help from his discord server
 - **overset** for [JP01](https://github.com/overset/JP01). The idea of publishing the case design on Onshape. After seeing their great work I switched from fusion360 to Onshape
 - And many community members for giving me advice and feedback, thanks!
+
+### JLCPCB Tutorial
+
+In order to print your PCB you will need three files which you can find here -> https://github.com/kkatano/bakeneko-65/releases
+
+- Gerber File which you upload as a zip
+- BOM File which includes all necessary electrical components (Capacitors, resistors, etc.)
+- CPL File
+
+Step 1:
+Quote Now
+
+Step 2: 
+You should select "Assemble Bottom side" because that is where all of the solderable components are. JLC PCB will not solder the JST connector so you need to get a JST-SH 4-pin cable.
+![JLC PCB Settings and SMT Settings](https://imgur.com/a/Wrdi3m2)
+
+Step 3: 
+Upload the BOM and CPL File
+![JLC PCB BOM and CPL File Upload](https://imgur.com/2hVYvp3)
+
```

Comment:
Please remove this part. I do not want to put detailed instructions on the top page of the project. I would like to enhance the documentation in the future, and I am currently considering how to structure it. I will use what you have written here as a reference at that time.